### ISSUE TITLE
AsyncDropdown is broken

### ISSUE THREAD

**Issue by braxtonmckee** [APrioriInvestments/object_database#73]
This doesn't work:

```
def cell(self):
    return cells.AsyncDropdown("Thisboo", lambda: cells.Text("this is inside"))
```

**Comment by darth-cheney** [comment]
@braxtonmckee I am addressing this, for the moment, over on my branch [`eric-issue-73`](https://github.com/APrioriInvestments/object_database/tree/eric-issue-73). It seems to be working fine now, and I've added some stuff that I think just got lost in all of the work we've been doing.
  
I'm not submitting a PR yet because I'd like to get the CellsDemo testing PR merged in first, then I can write a couple of small tests for this case. But if you want to pull in now in order to just use it (and forget about a PR), it should be working.

**Comment by darth-cheney** [comment]
@braxtonmckee See PR #77

**Comment by darth-cheney** [comment]
Closed by #77
